src/main/java/com/coveo/AbstractFMT.java

```diff
@@ -9,22 +9,21 @@
 import com.google.googlejavaformat.java.ImportOrderer;
 import com.google.googlejavaformat.java.RemoveUnusedImports;
 import com.google.googlejavaformat.java.RemoveUnusedImports.JavadocOnlyImports;
-import org.apache.maven.plugin.AbstractMojo;
-import org.apache.maven.plugin.MojoExecutionException;
-import org.apache.maven.plugin.MojoFailureException;
-import org.apache.maven.plugin.logging.Log;
-import org.apache.maven.plugins.annotations.Parameter;
-
 import java.io.File;
 import java.io.FileFilter;
 import java.io.IOException;
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import org.apache.maven.plugin.AbstractMojo;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugin.MojoFailureException;
+import org.apache.maven.plugin.logging.Log;
+import org.apache.maven.plugins.annotations.Parameter;
 
 public abstract class AbstractFMT extends AbstractMojo {
-  private Log logger = getLog();
-  private Formatter formatter = new Formatter();
+  protected Log logger = getLog();
```

Comment:
wondering if the logic for this could have been kept all in this class. The other implementation is deprecated anyway, and I don't see how it's a problem that it would suddenly log all the files that are indeed getting reformated.

Reply:
hmm I've done this to put logic only on `check` mojo 
Because I don't see why 'format' mojo would receive parameters ?

Reply:
you'll prefer if I add getter and keep attributes private ?

Reply:
Indeed, :yolo: